### PR TITLE
Upgrade JasperReports #630 Support Linux Environment #575

### PR DESCRIPTION
**Pull request by btyanagawamg**
Please review #630 #575 .

### CODE REVIEW

File: terasoluna-tourreservation-web/src/main/resources/IPAfont00303/Readme_IPAfont00303.txt [hunk 1]
```diff
@@ -0,0 +1,40 @@
+IPAフォント（IPAフォント4書体パック）
```
Comment (yoshikawaa) [inline terasoluna-tourreservation-web/src/main/resources/IPAfont00303/Readme_IPAfont00303.txt]: - [ ] Really is Readme_IPAfont00303.txt required?

Comment (yoshikawaa) [inline terasoluna-tourreservation-web/src/main/resources/IPAfont00303/Readme_IPAfont00303.txt]: - [ ] Really is IPAフォント4書体パック required?
You can also select IPAex明朝.

File: terasoluna-tourreservation-web/src/main/resources/IPAfont00303/fonts.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <normal>IPAfont00303/ipag.ttf</normal>
+    <pdfEncoding>Identity-H</pdfEncoding>
+    <pdfEmbedded>true</pdfEmbedded>
+  </fontFamily>
+  <fontFamily name="ipapg">
+    <normal>/IPAfont00303/ipagp.ttf</normal>
+    <pdfEncoding>Identity-H</pdfEncoding>
+    <pdfEmbedded>true</pdfEmbedded>
+  </fontFamily>
+  <fontFamily name="ipam">
```
Comment (yoshikawaa) [inline terasoluna-tourreservation-web/src/main/resources/IPAfont00303/fonts.xml]: - [ ] It seems better to rename `IPA明朝`.
See Readme_IPAfont00303.txt .

File: terasoluna-tourreservation-web/src/main/resources/IPAfont00303/fonts.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    <normal>IPAfont00303/ipam.ttf</normal>
+    <pdfEncoding>Identity-H</pdfEncoding>
+    <pdfEmbedded>true</pdfEmbedded>
+  </fontFamily>
+  <fontFamily name="ipapm">
```
Comment (yoshikawaa) [inline terasoluna-tourreservation-web/src/main/resources/IPAfont00303/fonts.xml]: - [ ] It seems better to rename `IPA P明朝`.
See Readme_IPAfont00303.txt .

File: terasoluna-tourreservation-web/src/main/resources/IPAfont00303/Readme_IPAfont00303.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/main/resources/IPAfont00303/Readme_IPAfont00303.txt]: fe94d974c74bf103a523626f860f0100d5dc8826

I upgrade in above commit .

Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/main/resources/IPAfont00303/Readme_IPAfont00303.txt]: fe94d974c74bf103a523626f860f0100d5dc8826

Not need, I changed to IPAex明朝 .

File: terasoluna-tourreservation-web/src/main/resources/IPAfont00303/fonts.xml (hunk 2 above)
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/main/resources/IPAfont00303/fonts.xml]: 5e56b3af6888784632042f7d7b6cdc11a88434e7

I fixed in above commit .

File: terasoluna-tourreservation-web/src/main/resources/IPAfont00303/fonts.xml (hunk 3 above)
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/main/resources/IPAfont00303/fonts.xml]: 5e56b3af6888784632042f7d7b6cdc11a88434e7

I fixed in above commit .

File: terasoluna-tourreservation-web/src/main/resources/IPAfont00303/Readme_IPAfont00303.txt (hunk 1 above)
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/main/resources/IPAfont00303/Readme_IPAfont00303.txt]: 22ade0bbfd4710d8e8c13e3e15ae4b0f0b8c7370

I removed in above commit .

**Pull request merged**